mkdocs.yml

```diff
@@ -118,6 +118,10 @@ plugins:
             filters: ["!^_.*$"]
             show_bases: false
             separate_signature: true
+          selection:
+            docstring_style: numpy
```

Comment:
@flaviuvadan could we use google docstyle? It is easier to write docs using it as we don't have to worry about underlining with hyphens and it follows python syntax more closely (colons and tabs).

Reply:
Yep, we can definitely do so! I am going to change the rest of the docs then